include/battle_controllers.h

```diff
@@ -124,8 +124,7 @@ struct ChooseMoveStruct
     u8 currentPp[MAX_MON_MOVES];
     u8 maxPp[MAX_MON_MOVES];
     u16 species;
-    u8 monType1;
-    u8 monType2;
+	u8 monTypes[2];
```

Comment:
Spaces instead of tabs.

Reply:
https://github.com/pret/pokeemerald/pull/2021/commits/82bad72ade1ee89c2bc63dc38087a5429d4a809f